app/src/main/java/com/teamltt/carcare/activity/DemoActivity.java

```diff
@@ -71,7 +85,7 @@ public void onReceive(Context context, Intent intent) {
             } else if (BluetoothAdapter.ACTION_DISCOVERY_FINISHED.equals(action)) {
                 Log.i("debug BT", "discovery unsuccessful");
```

Comment:
I see this String several times in this PR for use when logging. It should probably be a `static final String TAG` member of this class.